services/src/main/java/org/exoplatform/news/rest/NewsTargetsRestResourcesV1.java

```diff
@@ -0,0 +1,53 @@
+package org.exoplatform.news.rest;
+
+import io.swagger.annotations.*;
+import org.exoplatform.common.http.HTTPStatus;
+import org.exoplatform.services.log.ExoLogger;
+import org.exoplatform.services.log.Log;
+import org.exoplatform.services.rest.resource.ResourceContainer;
+import org.exoplatform.social.metadata.MetadataService;
+import org.exoplatform.social.metadata.model.Metadata;
+import org.exoplatform.social.metadata.model.MetadataKey;
+
+import javax.annotation.security.RolesAllowed;
+import javax.ws.rs.*;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriInfo;
+import java.util.List;
+
+@Path("v1/newsTargets")
+@Api(tags = "v1/newsTargets", value = "v1/newsTargets", description = "Managing news targets")
+public class NewsTargetsRestResourcesV1 implements ResourceContainer {
+
+  private static final Log       LOG                             = ExoLogger.getLogger(NewsTargetsRestResourcesV1.class);
+
+  private static final String       NEWS_TARGET_TYPE                             = "newsTarget";
+
+  private MetadataService metadataService;
+
+  public NewsTargetsRestResourcesV1(MetadataService metadataService) {
+    this.metadataService = metadataService;
+  }
+
+  @Path("getTargetsByType/{type}")
```

Comment:
to be deleted since v1/news/targeting will be the endpoint which will retrieve the list of targets